src/components/Menu/UserMenu/WalletWrongNetwork.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Button, Text } from '@pancakeswap/uikit'
+import { useTranslation } from 'contexts/Localization'
+
+const WalletWrongNetwork: React.FC = () => {
+  const { t } = useTranslation()
+
+  const handleOnClick = (): void => {
+    const url = 'https://docs.pancakeswap.finance/get-started/connection-guide'
+    window.open(url)
+  }
+
+  return (
+    <>
+      <Text mb="24px">{t('You’re connected to the wrong network.')}</Text>
+      <Button onClick={handleOnClick} mb="24px" variant="subtle">
```

Comment:
Can we use `Link` or `LinkExternal` components instead of `Button` in this case ?